recon_test_pack/old_scatter_cylinder.hs

```diff
@@ -0,0 +1,50 @@
+!INTERFILE  :=
+name of data file := old_scatter_cylinder.s
+originating system := unknown
+!version of keys := STIR3.0
```

Comment:
let's add `modality` and even `patient position` stuff

Reply:
Later you suggest to remove the file and so I did. I leave you to delete comment.